index.html

```diff
@@ -0,0 +1,32 @@
+
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="assets/styles/app.css">
+    <title>Document</title>
+</head>
+<body class="page">
+    <header class="header">
+        <img src="assets/icons/menu.svg" alt="Menu">
+    </header>
+    <section>
+        <div class="headline"> 
+            <h1 class="lines"> Hey there.</h1>
+            <h1 class="lines"> I'm THIERRY</h1>
+            <h1 class="middle lines">I'm a Software Developer</h1>
+            <p class="description"> I can help you build a product, feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, 
+                don’t hesitate to contact me.</p>
+            <div class="socials">
+              <img src="assets/icons/github.svg" alt="github">
+              <img src="assets/icons/linkeldn.svg" alt="linkeldn">
+              <img src="assets/icons/twitter.svg" alt="twitter">
+              <img src="assets/icons/angellist.svg" alt="angellist">
+              <img src="assets/icons/medium.svg" alt="medium">
+            </div>
```

Comment:
- [ ] Kindly consider taking a `ul` here instead of `div` and then each `img` tag would be first inside an `a` tag and then inside each `li`. this way we will be following best practices and best semantics.